apps/financial-aid/backend/src/app/modules/application/application.controller.ts

```diff
@@ -183,6 +185,9 @@ export class ApplicationController {
       throw new NotFoundException(`Application ${id} does not exist`)
     }
 
+    const staff = await this.staffService.findById(updatedApplication.staffId)
```

Comment:
Do we maybe want to have a check here if updateapplication has staffId then search for it, otherwise don't? 😸

Reply:
I am actually deleting this function in my next PR 🙈 

https://github.com/island-is/island.is/pull/5286